config/prow/Makefile

```diff
@@ -32,6 +32,7 @@ update-plugins: get-cluster-credentials
 	kubectl create configmap plugins --from-file=plugins.yaml=plugins.yaml --dry-run -o yaml | kubectl replace configmap plugins -f -
 
 deploy-prow: get-cluster-credentials
+	kubectl apply --server-side=true -f ./cluster/prowjob-crd/legacy/prowjob-schemaless_customresourcedefinition.yaml
```

Comment:
Are there any issues with using server-side apply for the schemaless CRD?

Reply:
should not be